Documentation/etcd-mixin/mixin.libsonnet

```diff
@@ -92,9 +92,9 @@
           {
             alert: 'etcdHighNumberOfFailedGRPCRequests',
             expr: |||
-              100 * sum(rate(grpc_server_handled_total{%(etcd_selector)s, grpc_code!="OK"}[5m])) without (grpc_type, grpc_code)
+              100 * sum(rate(grpc_server_handled_total{%(etcd_selector)s, grpc_code!="OK"}[1m])) without (grpc_type, grpc_code)
                 /
-              sum(rate(grpc_server_handled_total{%(etcd_selector)s}[5m])) without (grpc_type, grpc_code)
+              sum(rate(grpc_server_handled_total{%(etcd_selector)s}[1m])) without (grpc_type, grpc_code)
                 > 5
             ||| % $._config,
             'for': '5m',
```

Comment:
Does it mean that in worst case the service might by 80% of time unhealthy and the alert will not trigger ? 

4min of failures, 1 min below threshold, 4min of failures, ... 

Shell we decrease it to 2 or 3 min ?

Reply:
[marking as a requested change]